doc/manual/local.mk

```diff
@@ -24,7 +24,13 @@ dummy-env = env -i \
 	NIX_STATE_DIR=/dummy \
 	NIX_CONFIG='cores = 0'
 
-nix-eval = $(dummy-env) $(bindir)/nix eval --experimental-features nix-command -I nix/corepkgs=corepkgs --store dummy:// --impure --raw
+ifneq (,$(findstring qemu,$(CROSS_EMULATOR)))
+  _nix-emu := QEMU_LD_PREFIX=${QEMU_LD_PREFIX} $(CROSS_EMULATOR)
```

Comment:
if QEMU_LD_PREFIX is exported, shouldn't it just pass it through to CROSS_EMULATOR?

Reply:
It will be unset if qemu was called with dummy-env.

Reply:
ah makes sense